### ISSUE TITLE
Getting error about 'color' type

### ISSUE THREAD

**Issue by micarner** [AndreBClark/cosmic-schemas#9]
Hello,
I have installed the configured this plugin like below:

```
export default defineConfig({
  basePath: '/studio',
  projectId,
  dataset,
  title,
  schema: {
    // If you want more content types, you can add them to this array
    types: [authorType, postType, settingsType, vanForSale,
      vansForSaleSettingsType,homePageSettingsType,
      openGraph,siteMeta, manifest, meta, siteSettings, locale, googleAnalytics,
    ]
  },
  plugins: [
    singletonTools(),
    structureTool({
      structure: settingsStructure(settingsType),
      // `defaultDocumentNode` is responsible for adding a “Preview” tab to the document pane
      defaultDocumentNode: previewDocumentNode()
    }),
    presentationTool({
      locate,
      previewUrl: {
        draftMode: {
          enable: DRAFT_MODE_ROUTE
        }
      }
    }),
    // Configures the global "new document" button, and document actions, to suit the Settings document singleton
    settingsPlugin({ type: settingsType.name }),
    // Add an image asset source for Unsplash
    unsplashImageAsset(),
    // The remaining plugins are only loaded in dev mode
    process.env.NODE_ENV !== 'production' && debugSecrets(),
    // Vision lets you query your content with GROQ in the studio
    // https://www.sanity.io/docs/the-vision-plugin
    process.env.NODE_ENV !== 'production' &&
    visionTool({ defaultApiVersion: apiVersion }),
  ]
})
```

However, when I try to visit the studio, I get this error:

![image](https://github.com/AndreBClark/cosmic-schemas/assets/20423592/2860b57d-46a1-44c5-b857-dd5397af515d)

It looks like there is a 'color' type in your code, but I am unsure what type that is supposed to be:

![image](https://github.com/AndreBClark/cosmic-schemas/assets/20423592/0026a6e2-a035-42ea-b9fb-fadc87d3b26e)

**Comment by AndreBClark** [comment]
@micarner hi! what did you end up doing to resolve your issue? how can i prevent others from having this issue as well?

**Comment by AndreBClark** [comment]
fixed by 64992cea4359e011e245728b8646288dd03c1d14 
color fields now expect a regular string

**Comment by micarner** [comment]
Thanks for the update. It took me some searching, but it was needing the colorInput() plugin.
